### PR TITLE
Login Responsiveness Fixed

### PR DESCRIPTION
**Pull request by arpit456jain**
# Changes
I have made the login page responsive 

 Fixes : #101 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

https://user-images.githubusercontent.com/55352601/116369566-27366800-a827-11eb-8e07-1ababd3dc33b.mp4

### CODE REVIEW

File: pages/login.php [hunk 1]
```diff
@@ -1,22 +1,19 @@
 <?php
```
Comment (siv2r) [inline pages/login.php:1]: this empty PHP tag can be removed

File: pages/login.php [hunk 2]
```diff
@@ -113,18 +118,22 @@
 
 			<div class="form-elements">
           <img src="../images/red-avatar.png" alt="Avatar" class="avatar">
-          <?php 
-            if(isset($_GET["error"])) {
-              if ($_GET["error"] == "emptyInputLogin") {
+          <?php
+
+            if ( isset( $_GET["error"] ) ) {
```
Comment (siv2r) [inline pages/login.php:123]: did you apply the `phpfmt` format? That is not required, just writing a readable code will do

File: pages/login.php [hunk 3]
```diff
@@ -141,10 +150,12 @@
...
 
 
-			
+
 	</form>
-  <script src="../js/showpassword.js"></script>  
-<?php include("../include/footer.inc.php") ?>
+</div>
+  <script src="../js/showpassword.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js" integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf" crossorigin="anonymous"></script>
```
Comment (siv2r) [inline pages/login.php:157]: why are there **two CDN** links? Aren't they the same?

**Review by siv2r — CHANGES_REQUESTED** [review]

File: pages/login.php (hunk 2 above)
Comment (arpit456jain) [reply pages/login.php:123]: I think its because of format on save. Ok no problem i will remove the phpfmt settings.

File: pages/login.php (hunk 3 above)
Comment (arpit456jain) [reply pages/login.php:157]: I think there is only 1 cdn link 😅 other one is js for show password

File: pages/login.php (hunk 2 above)
Comment (siv2r) [reply pages/login.php:123]: Ahh..okay

File: pages/login.php (hunk 3 above)
Comment (siv2r) [reply pages/login.php:157]: I saw another one on top of the file.

Comment (siv2r) [reply pages/login.php:157]: Wait, I will tag there.

File: pages/login.php [hunk 4]
```diff
@@ -1,22 +1,19 @@
 <?php
 
-//some php code for verifying the user
-
-?>
+  //some php code for verifying the user
 
+;?>
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
```
Comment (siv2r) [inline pages/login.php:6]: This one
Is this link included again at the bottom?

Comment (arpit456jain) [reply pages/login.php:2]: no .

The link at the top is for css and at bottom its for js

Comment (siv2r) [reply pages/login.php:2]: okay

**Comment by siv2r** [comment]
Is the PR Complete? 
Everything looks good to merge

**Comment by arpit456jain** [comment]
> Is the PR Complete?
> Everything looks good to merge

yup its complete from my side i can send a video for confirmations

**Comment by arpit456jain** [comment]
mobile
dk

**Comment by siv2r** [comment]
Looks great!! Nicely done :tada: :tada:

**Comment by arpit456jain** [comment]
> Looks great!! Nicely done 🎉 🎉

thanks ...

Should i create a new issue for register page before making PR for it

**Pull request merged**